sdks/go/build.gradle

```diff
@@ -21,13 +21,46 @@ applyGoNature()
 
 description = "Apache Beam :: SDKs :: Go"
 
+def getLocalPlatform = {
+  String hostOs = com.github.blindpirate.gogradle.crossplatform.Os.getHostOs()
+  String hostArch = com.github.blindpirate.gogradle.crossplatform.Arch.getHostArch()
+  return hostOs + '-' + hostArch
+}
+
 golang {
   packagePath = 'github.com/apache/beam/sdks/go'
+
   build {
-    go 'build -o ./build/bin/beamctl github.com/apache/beam/sdks/go/cmd/beamctl'
+    // We always want to build linux-amd64 in addition to the user host platform
+    // so we can submit this as the remote binary used within the Go container.
+    //
+    // TODO: Move this into build_rules.gradle applyGoNature. This currently
+    // can't be done because com.github.blindpirate.gogradle.crossplatform.*
+    // is not found within when invoked from within build_rules.gradle applyGoNature
+    targetPlatform = [getLocalPlatform(), 'linux-amd64']
 
     // The symlinks makes it hard (impossible?) to do a wildcard build
     // of pkg. Go build refuses to follow symlinks. Drop for now. The files
     // are built when tested anyway.
```

Comment:
What is this comment referring to?

Reply:
Referring 'go build ./...' which builds all